dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -67,6 +67,7 @@
 
   static ClassDumper create(List<Path> jarFilePaths) throws IOException, ClassNotFoundException {
     // Creates classpath in the same order as inputClasspath for BCEL API
+        
```

Comment:
Why this change?

Reply:
random junk as I edited

Reply:
Reverting a file is a much more onerous operation in git than perforce. I also wish we could configure Github like we do critique to not even show diffs in trailing whitespace.